clang/tools/clang-linker-wrapper/LinkerWrapperOpts.td

```diff
@@ -184,6 +184,14 @@ def sycl_module_split_mode_EQ :
   Flags<[WrapperOnlyOption]>,
   HelpText<"Option that turns on split library with the given split mode">;
 
+def use_sycl_post_link_tool : Flag<["--", "-"], "use-sycl-post-link-tool">,
+  Flags<[WrapperOnlyOption]>,
+  HelpText<"Use sycl-post-link tool. Turned on by default. Option will be removed when sycl-post-link tool becomes removed.">;
+
+def no_use_sycl_post_link_tool : Flag<["--", "-"], "no-use-sycl-post-link-tool">,
+  Flags<[WrapperOnlyOption]>,
+  HelpText<"Use SYCL library instead of sycl-post-link tool. Only for testing purposes.">;
```

Comment:
Sorry does "Only for testing purposes." here mean that it is not ready to use, rather then it will never be on by default? If so, maybe we use the `EXPERIMENTAL` wording instead

Reply:
> Does "Only for testing purposes." here mean that it is not ready to use?

At the moment, sycl-post-link can't be replaced by the Library. Once the library is ready this option will be turned on by default.

Reply:
got it, the description is correct as-is then imo